lib/feedsub.js

```diff
@@ -47,16 +48,29 @@ module.exports = class FeedReader extends EventEmitter {
       }
     });
 
+    /* preload history if available */
     this.news.addHistory('item', this.options.history);
 
     this.newitems = [];
     this.first = this.news.history.get('item').size === 0;
+    this.initialEmitCount = 0;
```

Comment:
doesn't need to be part of `this` since it doesn't have to be exposed to the FeedSub object. can use a `let`. it can also be a boolean since it's only ever `0` or `1`